packit_service/worker/handlers/abstract.py

```diff
@@ -209,6 +211,49 @@ def get_packit_commands_from_comment(
     return []
 
 
+class CeleryTask:
```

Comment:
My initial thought was to put the relevant methods into `JobHandler` but I like your solution (new class wrapping celery's `Task`) more :+1: